source/Sashimi.Terraform.Tests/ActionHandlersFixture.cs

```diff
@@ -363,17 +368,25 @@ void PopulateVariables(TestActionHandlerContext<Program> _)
             output = ExecuteAndReturnResult(typeof(TerraformApplyActionHandler), PopulateVariables, "Azure");
             output.OutputVariables.ContainsKey("TerraformValueOutputs[url]").Should().BeTrue();
             output.OutputVariables["TerraformValueOutputs[url]"].Value.Should().Be(expectedHostName);
-            await MakeRequest();
+            await AssertRequestResponse(HttpStatusCode.Forbidden);
 
             ExecuteAndReturnResult(typeof(TerraformDestroyActionHandler), PopulateVariables, "Azure");
-            Func<Task> request = async () => await MakeRequest();
-            request.Should().Throw<HttpRequestException>().WithMessage("No such host is known.");
 
-            async Task MakeRequest()
+            //This will throw on some platforms and return "NotFound" on others
+            try
+            {
+                await AssertRequestResponse(HttpStatusCode.NotFound);
+            }
+            catch (HttpRequestException ex)
+            {
+                ex.Message.Should().Contain("known");
```

Comment:
Should this be "No such host is known." ?

Reply:
On some platforms you get a different error message, I can dig up the other one and assert that it's one of the 2?

Reply:
So how about `await AssertResponseIsNotReachable();` and this checks for 404 or "No such host is known."